system/testing/BaseTestCase.cfc

```diff
@@ -311,15 +311,17 @@ component extends="testbox.system.compat.framework.TestCase" accessors="true"{
 	/**
 	* Executes a framework lifecycle by executing an event.  This method returns a request context object that can be used for assertions
 	* @event The event to execute
+    * @url The url to execute
 	* @private Call a private event or not
 	* @prePostExempt If true, pre/post handlers will not be fired.
 	* @eventArguments A collection of arguments to passthrough to the calling event handler method
 	* @renderResults If true, then it will try to do the normal rendering procedures and store the rendered content in the RC as cbox_rendered_content
-	* 
+	*
 	* @return coldbox.system.context.RequestContext
 	*/
 	function execute(
-		required event,
+		string event = "",
+        string url = "",
```

Comment:
Maybe this should be `URI` instead of `URL` as it technically it is a resource identifier not a link?